package-readmes/storesShareasale.md

```diff
@@ -0,0 +1,156 @@
+## ShareASale Integration 
+
+The ShareASale Integration package provides easy integration with your ShareASale account. You can track purchases made on your site through affiliate links, and manage commissions for your affiliates.
+
+
+### Setup 
+
+Before using the package, set up the following:
+
+
+#### ShareASale Platform 
+
+
+1. Create a ShareASale [Merchant Account](https://account.shareasale.com/merchant/signup.cfm), or login if you have an existing account.
+2. In your dashboard on the left corner you should see your ShareASale Merchant ID number following “ID#”. 
+3. Go to **Tools > Merchant API** to get your ShareASale **Token** and **API Secret**.
+
+
+#### Wix Platform 
+
+
+
+*   Add a **Wix Store** to your site.
+*   Open the [Wix Secrets Manager](https://support.wix.com/en/article/velo-about-the-secrets-manager), and store your Merchant ID, ShareASale token, and API secret as new secrets with the following names: 
+
+	`velo-shareasale-merchantId`: YOUR_SHAREASALE_MERCHANT_ID
+
+ 	`velo-shareasale-token`: YOUR_SHAREASALE_TOKEN
+
+ 	`velo-shareasale-apiSecret`: YOUR_SHAREASALE_API_SECRET
```

Comment:
I don't think you need to snake case here, since it's not an example in code. Should also help with readability.

```suggestion
	`velo-shareasale-merchantId`: Your ShareASale merchant ID

 	`velo-shareasale-token`: Your ShareASale token

 	`velo-shareasale-apiSecret`: Your ShareASale API secret
```